src/carma-streets/parse_ros_bags.py

```diff
@@ -37,6 +37,52 @@ def get_detected_objects(ros_bag_file, outputfile, time_offset):
     print(f"Calculated sim time offset: '{time_offset}'")
     
 
+def get_pedestrian_odometry(ros_bag_file, output_file, time_offset):
```

Comment:
The script looks like it can work for any odometry given the id but at the same time the topic is tied to carla.
So maybe we can rename to `get_carla_actor_odometry` or something like that and receive the id as input